modules/post/windows/gather/enum_hyperv_vms.rb

```diff
@@ -30,11 +35,11 @@ def run
     unless have_powershell?
       fail_with(Failure::NoAccess, "The target does not have PowerShell installed so we can't access the state of the Hyper-V VMs")
     end
-    results = psh_exec('Get-VM')
-    if results =~ /is not recognized as the name of a cmdlet/
-      print_error('The target is not a Hyper-V host')
-    elsif results =~ /do not have the required permission/
-      print_error('You need to be running as an elevated admin or a user of the Hyper-V Administrators group to run this module')
+    get_vm = "try { Get-VM } catch {echo #{error_token}; echo $Error[0]}"
+    results = psh_exec(get_vm)
+    if results =~ /#{error_token}/
+      results.delete!(error_token)
+      print_error("Error running `Get-VM` command. \n #{results} ")
```

Comment:
Instead of concatenating it with a newline, you can actually use `print_line` to print the results without the status indicator that's prepended by `print_good`, `print_status`, etc.

```suggestion
      print_error('Error running `Get-VM` command:')
      print_line(results)
```

Reply:
Committed the changes! :+1: